lambda/src/main/resources/db/migration/V8__add-matches-table

```diff
@@ -0,0 +1,38 @@
+DROP TABLE "FFIDMetadata";
```

Comment:
Just to check - it's not going to affect anyone if we lose all the metadata that's in here already?

Reply:
It seems unlikely because this will be run into the database before anyone starts using it but just in case, I've added in some sql to move any existing data across. That way, if there is anything we'll keep it and if there isn't it'll do nothing.